docs/source/conf.py

```diff
@@ -166,6 +166,7 @@
     r"https://arxiv.org/abs/1807.05572",
 ]
 
+suppress_warnings = ["myst.header"]
```

Comment:
Added this due to the following warning when I checked the build locally. I am open to other suggestions. Right now, the warnings like this are supressed for all files in the build. 

```
/mitiq/docs/source/apidoc.md:199: WARNING: Non-consecutive header level increase; H2 to H4 [myst.header]                                                                         
/mitiq/docs/source/apidoc.md:201: WARNING: Non-consecutive header level increase; H2 to H4 [myst.header]
/mitiq/docs/source/apidoc.md:207: WARNING: Non-consecutive header level increase; H2 to H4 [myst.header]
/mitiq/docs/source/apidoc.md:215: WARNING: Non-consecutive header level increase; H2 to H4 [myst.header]
/mitiq/docs/source/apidoc.md:221: WARNING: Non-consecutive header level increase; H2 to H4 [myst.header]
/mitiq/docs/source/apidoc.md:228: WARNING: Non-consecutive header level increase; H2 to H4 [myst.header]
/mitiq/docs/source/apidoc.md:235: WARNING: Non-consecutive header level increase; H2 to H4 [myst.header]
/mitiq/docs/source/apidoc.md:242: WARNING: Non-consecutive header level increase; H2 to H4 [myst.header]
/mitiq/docs/source/apidoc.md:250: WARNING: Non-consecutive header level increase; H2 to H4 [myst.header]

```

Reply:
Actually never mind. The warning was raised due to a mistake I made.